### ISSUE TITLE
how to get the .bin file?

### ISSUE THREAD

**Issue by PSlearner** [Mintzh/MobilenetOnZynq#1]
Hello, I have 3 questions:
(1) the convolution kernel in the orginal MobileNet only hava weights , how to get the bias.bin like in your wbin folder?
(2) how to run the batchnormalization layer on the Zynq? are you transferring the BN layer to the bias.bin?
(3) what framework do you use? TF or Pytorch or Caffe? how to convert the saved model to the .bin file.
Thanks a lot! @Mintzh

**Comment by Mintzh** [comment]
Hello, 
Firstly the framework is in TF, and this project is for the implemention of this demo on Zynq for my course project：
https://github.com/rwightman/posenet-python
1) the weight .bin is just the weight in NHWC order to be saved in binary with numpy
2) it seems that the network in the demo do not have batchnorm layer, and I think your guess may be right
For acceleration, I use Zynq to do most of the pointwise convolution, if you are interested in the whole project, please feel free to contact me.

**Comment by PSlearner** [comment]
I will see the project you mentioned. Thank you a lot!
